src/main/java/org/aksw/gerbil/evaluate/ExtendedMacros.java

```diff
@@ -0,0 +1,39 @@
+package org.aksw.gerbil.evaluate;
+
+import org.aksw.gerbil.matching.EvaluationCounts;
+
```

Comment:
Please add a Javadoc comments to the class, attributes and methods.

Reply:
Added Javadoc comments